### ISSUE TITLE
Accessibility: Linked images using CSS class "img-responsive" do not have focus indicator

### ISSUE THREAD

**Issue by aduth** [GSA/centers-of-excellence#358]
For linked images where the image is assigned CSS class `img-responsive`, there is no visible indication when the link is focused.

**Steps to Reproduce:**

1. Navigate to https://coe.gsa.gov/about/approach-team-structure.html
2. <kbd>Tab</kbd> to navigate to the linked image under "How We Work"

_Expected:_ Focus should be visible.
_Actual:_ Focus is not visible.

**WCAG Success Criteria (AA):** [2.4.7: Focus Visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html)

**Relevant technique:** [CSS C15: Using CSS to change the presentation of a user interface component when it receives focus](https://www.w3.org/WAI/WCAG21/Techniques/css/C15)

**Relevant failure:** [F78: Failure of Success Criterion 2.4.7 due to styling element outlines and borders in a way that removes or renders non-visible the visual focus indicator](https://www.w3.org/WAI/WCAG21/Techniques/failures/F78)

**Remediation guidance:**

Focus styling appears to be absent due to assignment of `display: block` on images of class name `img-responsive`.
